src/locale/locales/ro/messages.po

```diff
@@ -212,122 +217,122 @@ msgstr "{count} elemente necitite"
 #: src/lib/generate-starterpack.ts:108
 #: src/screens/StarterPack/Wizard/index.tsx:178
 msgid "{displayName}'s Starter Pack"
-msgstr "Pachetul de început al {displayName}"
+msgstr "Pachetul de pornire al lui {displayName}"
```

Comment:
It was decided in https://github.com/bluesky-social/social-app/pull/6456#issuecomment-2498121422 to use "pachet de început". See the [glossary](https://github.com/bluesky-social/social-app/pull/6456#issue-2665899849)